gulp-tasks/build.js

```diff
@@ -44,11 +45,58 @@ const buildPackage = (projectPath) => {
 
   return fse.emptyDir(buildDir)
     .then(() => projectBuildProcess())
-    .then(() => fse.copy(path.join(__dirname, '..', 'LICENSE'),
-      path.join(projectPath, 'LICENSE')))
+    .then(() => buildTestBundles(projectPath))
+    .then(() => {
+      fse.copy(path.join(__dirname, '..', 'LICENSE'),
+          path.join(projectPath, 'LICENSE'));
+    })
     .then(() => printBuildTime(`${(Date.now() - startTime) / 1000}s`));
 };
 
+/**
+ * Builds the browser and service worker test bundles.
+ * @param {String} projectPath The path to a project directory.
+ * @return {Promise} Resolves if building succeeds, rejects if it fails.
+ */
+const buildTestBundles = (projectPath) => {
+  const getPlugins = () => [
+    // multiEntry uses glob so it requires `/` separators in paths.
+    multiEntry({exports: false}),
+    resolve({jsnext: true, main: true, browser: true}),
+    commonjs(),
+  ];
+
+  const project = path.basename(projectPath);
+
+  return Promise.all([
+    buildJSBundle({
+      rollupConfig: {
+        entry: `./packages/${project}/test/browser/*.js`,
+        plugins: getPlugins(),
+      },
+      writeConfig: {
+        sourceMap: true,
+        format: 'iife',
+        dest: `./packages/${project}/build/test/browser.js`,
```

Comment:
Nit: Could we call this `browser-bundle.js` just so it hints that this isn't the source.